scripts/gke-allowlist/refresh-digests.py

```
@@ -0,0 +1,129 @@
+
+REPO = os.path.expanduser("~/projects/workspace-gke-autopilot-approval/gke-allow-list/ARMO")
+DEFAULT_FILES = sorted(glob.glob(f"{REPO}/*/1.40/*-1.40.yaml"))
+TAG_RE = re.compile(r"^v\d+\.\d+\.\d+$")
+REG_RE = re.compile(r"quay\.io/(armosec|kubescape)/node-agent")
+
+
+@lru_cache(maxsize=None)
+def latest(registry, maxn):
+    """Return list of (tag, digest) for the latest `maxn` vX.Y.Z tags, newest first."""
+    repo = f"quay.io/{registry}/node-agent"
+    tags = subprocess.run(["skopeo", "list-tags", f"docker://{repo}"],
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)